src/components/Icon.js

```diff
@@ -1,11 +1,7 @@
 import React from 'react'
 
-const Icon = props => {
-    const {
-        onclick,
-        id,
-    } = props;
-    return <div className="toolbar-icon" onClick={ ()=>onclick(id)} >
+const Icon = ({ onclick, id }) => {
+    return <div className="toolbar-icon" onClick={ () => onclick(id)} >
         <svg version="1.1" className="icon" xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 19.481 19.481" enableBackground="new 0 0 19.481 19.481">
```

Comment:
Can you kindly store the SVG on each shortcut object in `/src/shortcuts/`? They will end up being specific to each shortcut, and that will make it easy to add new shortcuts without updating the Toolbar. Thank you!

Reply:
Sure thing.